packages/gatsby-mdx/page-with-mdx.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
```

Comment:
replaces `componentWithMDXScope`.

Create the wrapper file for pages with MDX queried through GraphQL.